symengine/lib/symengine_wrapper.pyx

```diff
@@ -689,7 +689,7 @@ cdef class Basic(object):
         elif (op == 5):
             return c2py(<RCP[const symengine.Basic]>(symengine.Ge(A.thisptr, B.thisptr)))
 
-    def expand(Basic self not None):
+    def expand(Basic self not None, *args, **kwargs):
```

Comment:
As I mentioned we should implement them. Accepting any argument and doing nothing is worse than throwing an error, because it may suggest to the user that it is allowed.